apps/heft/src/plugins/CopyStaticAssetsPlugin.ts

```diff
@@ -16,6 +16,7 @@ import { IBuildStageContext, ICompileSubstage } from '../stages/BuildStage';
 import { ScopedLogger } from '../pluginFramework/logging/ScopedLogger';
 import { CoreConfigFiles } from '../utilities/CoreConfigFiles';
 import { RigConfig } from '@rushstack/rig-package';
+import { ITypeScriptConfigurationJson } from './TypeScriptPlugin/TypeScriptPlugin';
 
 const PLUGIN_NAME: string = 'CopyStaticAssetsPlugin';
```

Comment:
Seems like `CopyStaticAssetsPlugin` should just be part of the TypeScript plugin now. If it's a nontrivial change, don't worry about it now.

Reply:
I think keeping them separate is cleaner. They're different operations, even if they appear the same to the developer.